packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts

```diff
@@ -0,0 +1,111 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { BalanceTransferPayload, TransferDomainType } from '../../../src/category/account'
+import { RpcApiError } from '@defichain/jellyfish-api-core/dist/index'
+import BigNumber from 'bignumber.js'
+
+describe('TransferDomain', () => {
+  let dfiAddress: string, ethAddress: string
+  const amountToTransfer = 5
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForWalletCoinbaseMaturity()
+    await client.masternode.setGov({
+      ATTRIBUTES: {
+        'v0/params/feature/evm': 'true'
+      }
+    })
+    await container.generate(1)
+    dfiAddress = await container.call('getnewaddress')
+    // transfer utxostoaccount
+    await container.call('utxostoaccount', [{ [dfiAddress]: '100@0' }])
+    await container.generate(1)
+    ethAddress = await container.getNewAddress('', 'eth')
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
```

Comment:
Let's add a few more fine-grained tests if the data is invalid e.g wrong type, invalid address, Not enough balance to cover in ethAddress to cover the transfer, - [reference](https://github.com/JellyfishSDK/jellyfish/blob/main/packages/jellyfish-api-core/__tests__/category/token/burnTokens.test.ts#L71-L82). Some of the [transferDomain tests](https://github.com/DeFiCh/ain/blob/feature/evm/test/functional/feature_evm.py#L94-L105) on the ain side